scapy/contrib/coap.py

```diff
@@ -235,6 +278,12 @@ def post_dissect(self, pay):
                 self.content_format = k[1]
         return pay
 
+    def hashret(self):
+        return struct.pack('I', self.msg_id) + self.token
+
+    def answers(self, other):
```

Comment:
Wouldn't it make sense to implement the answers function based on coap_codes / the field "code"

Reply:
I added a simple verification where any kind of answer is accepted (within the response codes in the RFC's section 5.9), except if you are trying to answer with another request.

I did this way because I don't want to put too much constraints for the user.